doc/index.md

```diff
@@ -41,6 +41,10 @@ Preferences
 
         <preference name="StatusBarBackgroundColor" value="#000000" />
 
+- __AndroidStatusBarBackgroundColor__ (color hex string, defaults to the Android theme default). On Android 5 and up, the background color can be set by a hex string (#RRGGBB) at startup. We don't use the same property as for iOS because on iOS you typically want the statusbar to have the same color as the app background, but the Android 5+ guidelines specify using a different color than you apps main color, so the value of this property is typically different than the one specified by StatusBarBackgroundColor.
```

Comment:
@sinofool That would be a great addition to the spec, but unfortunately that's not how it seems to work. The platform ignores the `platform` tag and simply picks up the last value of a preference you specify.

So this:

```xml
    <platform name="android">
        <preference name="StatusBarBackgroundColor" value="#444444" />
    </platform>

    <platform name="ios">
        <preference name="StatusBarBackgroundColor" value="#cccccc" />
    </platform>
```

means `#cccccc` is applied to all platforms.